thoth/common/exceptions.py

```diff
@@ -17,29 +17,29 @@
 """Exceptions used within thoth-common package."""
 
 
-class ThothCommonException(Exception):
+class ThothCommonExceptionError(Exception):
```

Comment:
why were the name changed ThothCommonExceptionError ? 
Exception in code itself is error
,Now,
right ?😅

Reply:
https://prow.operate-first.cloud/view/s3/ci-prow-artifacts--e2081a4d-fd23-4347-93d5-f33e0604d95d/prow-logs/pr-logs/pull/thoth-station_common/1186/pre-commit/1413135455687282688 pre-commit is not happy apparentl :/

Reply:
ah cool thnaks for poiniting this out.
```
flake8...................................................................Failed
- hook id: flake8
- exit code: 1
thoth/common/exceptions.py:20:8: N818 exception name 'ThothCommonException' should be named with an Error suffix
thoth/common/exceptions.py:24:8: N818 exception name 'NotFoundException' should be named with an Error suffix
thoth/common/exceptions.py:36:8: N818 exception name 'NotKnownThothIntegration' should be named with an Error suffix
thoth/common/exceptions.py:40:8: N818 exception name 'KebechetInputsMissing' should be named with an Error suffix
```